src/hpc/tls.h

```diff
@@ -1,12 +1,28 @@
 #ifndef GAP_TLS_H
 #define GAP_TLS_H
 
+#if !defined(HPCGAP)
+
+/*
+ * HPC-GAP stubs.
+ */
+
+#define ReadGuard(bag) NOOP
+#define WriteGuard(bag) NOOP
+
+static inline Bag ImpliedWriteGuard(Bag bag)
+{
+  return bag;
+}
+
+#else
+
 #include <stdint.h>
 
-#include "code.h"
+#include "../code.h"
 #include "tlsconfig.h"
-#include "scanner.h"
-#include "gasman.h"
+#include "../scanner.h"
+#include "../gasman.h"
```

Comment:
Aha, now I see what you mean with "relative paths". Well, that is slightly ugly. But still not a show stopper -- if we can't fix it now, we certainly will be able to fix it later with the new build system. In any case, this is a minor detail, and not a showstopper.